.github/ISSUE_TEMPLATE/Security_issue.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Security issue
+about: Report security issue affecting this project
+
+---
+
+A clear and concise description of how the security issue affects this project
+
+Things to consider
+ * Severity?
+ * Is in production?
+ * Affected components?
+
+## Steps to reproduce
+
+Not always applicable
+ 1. Step 1
+ 2. Step 2
+ 3. Step ...
+
+## Migitation
+
+Migitation options, if there are any
+
+## Additional resources
```

Comment:
Let's rename this to "Vulnerability Information" and move it before "Steps to reproduce"